src/framework/audio/engine/internal/audioengine.cpp

```
@@ -85,15 +86,41 @@ void AudioEngine::deinit()
+        return it->second;
     }
+    auto context = std::make_shared<AudioContext>(ctxId);
+    m_contexts[ctxId] = context;
+    return context;
+*/
+}
+
+void AudioEngine::destroyContext(const modularity::IoCID& ctxId)
+{
+    UNUSED(ctxId);
+/*
```

Cppcheck (2.20.0)

[style] The function 'setInputParams' is never used.

(unusedFunction)